getstarted.rst

```diff
@@ -64,31 +62,53 @@ Each nio project will be in its own directory too. We will start by creating som
     mkdir projects
     mkdir versions
 
-Installation
+Global Installation (no virtual environment)
 ------------
 
-Lets start by installing nio version 1.5.1.
+Let's start by installing nio version 1.5.3 globally (without the use of a python virtual environment). In most cases, this is the method you will want to use. If you're installing n.io onto a device where you'll need the ability to switch between different versions for development/testing purposes, you may want to skip ahead to the intructions for installing with a **virtual environment**, but for almost all other cases, a global install is the way to go.
+
+First we need to make sure the our user has the proper permissions to install python packages via ``pip``. This is done by adding our user to a group that has access to ``/usr/local/lib/python3.4/site-packages``. This group is usually ``staff`` on most linux distributions. The command to do this would be:
+
+.. code-block:: bash
+   
+    sudo usermod -a -G staff <username>
+    
+
+Now we need to get the wheels for **nio** and **nioext** onto the device which n.io will be installed to. Once the wheels are on the device, navigate to the directory they have been copied to, and install them with these commands:
+
+.. code-block:: bash
+
+    pip install nio-1.5.3-py3-none-any.whl
```

Comment:
Update this to pip3.4 to be safe. On most machines, just pip usually point to 2.7